Sweeper/View Controllers/Home/PinsMapViewController.swift

```diff
@@ -125,9 +128,17 @@ class PinsMapViewController: UIViewController, NVActivityIndicatorViewable {
         locationManager.delegate = self
         locationManager.desiredAccuracy = kCLLocationAccuracyBest
         locationManager.distanceFilter = 200
-        locationManager.requestWhenInUseAuthorization()
+        locationManager.requestAlwaysAuthorization()
         locationManager.startUpdatingLocation()
     }
+    
+    private func requestNotificationsPermission() {
+        AppService.sharedInstance.requestLocalNotificationsPermissions {
+            let alertController = UIAlertController(title: nil, message: "You can activate notifications from Settings at a later time", preferredStyle: .alert)
```

Comment:
Can we use the global alert view for this? https://github.com/carnivorous-fuzz/pin-drop/blob/master/Sweeper/Views/DialogBox.swift

Reply:
```
let button = Dialog.button(title: "Try Again", type: .plain, action: nil)
Dialog.show(controller: self, title: "Unable to load pins", message: error?.localizedDescription ?? "Error", buttons: [button], image: nil, dismissAfter: nil, completion: nil)
```